test/arrayops.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+using Test
```

Comment:
Enables: `./julia test/arrayops.jl` to execute
(without having to run through `make`, which might force a rebuild like `make test-arrayops`)